UI/src/model/selectedLaneModelling.js

```diff
@@ -334,14 +334,7 @@
 
     this.removeLane = function(laneNumber, marker) {
       var laneIndex = getLaneIndex(laneNumber, marker);
-      var removedLane = self.selection.splice(laneIndex, 1)[0];
-
-      if (removedLane.id !== 0) {
-        //removedLane could be modified by the user so we need to fetch the original
-        var originalRemovedLane = _.find(lanesFetched, {'id': removedLane.id});
-        originalRemovedLane.isDeleted = true;
-        assetsToBeRemoved.push(originalRemovedLane);
-      }
+      self.selection.splice(laneIndex,1);
```

Comment:
Is this ok ? 
Do a slice without saving the result?

Reply:
We do not need to save the result. Now the delete will work only on lanes that are yet to be created(user is adding and then deleting the lane withou saving it first) so there is no need to send the info to the backend

Reply:
So this slice is useless because nothing will be done with the result and the selection will remaining  unchanged.

Reply:
No, the splice will alter the array it spliced.

Reply:
My mistake.